code_schemes/rqas/porticus/porticus_s01e05.json

```
@@ -3,6 +3,54 @@
     "Name": "Porticus S01E05",
     "Version": "0.0.0.1",
     "Codes": [
+        {
+            "CodeID": "code-03db8eb3",
+            "DisplayText": "equal job opportunities",
+            "CodeType": "Normal",
+            "NumericValue": 1,
+            "StringValue": "equal_job_opportunities",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-9338d6cc",
+            "DisplayText": "education and capacity building",
+            "CodeType": "Normal",
+            "NumericValue": 2,
+            "StringValue": "education_and_capacity_building",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-883bb831",
+            "DisplayText": "financial assistance",
+            "CodeType": "Normal",
+            "NumericValue": 3,
+            "StringValue": "financial_assistance",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-fb43c2ed",
+            "DisplayText": "invove them in decision making",
```

```suggestion
            "DisplayText": "involve them in decision making",
```
same typo in string value

---

Thanks, updated.